numba/cuda/cuda_paths.py

```diff
@@ -118,6 +144,55 @@ def get_conda_ctk():
     return os.path.dirname(max(paths))
 
 
+def get_nvidia_nvvm_ctk():
```

Comment:
Is there benefit in memozing all these `get` functions with `functools.cache`?

Reply:
As per above, it seems to make no difference.